manifests/config.pp

```diff
@@ -186,9 +187,12 @@
         systemd::unit_file {'prometheus.service':
           content => template('prometheus/prometheus.systemd.erb'),
         }
+        -> Class['systemd::systemctl::daemon_reload']
```

Comment:
ah good catch. the daemon reload needs to happen before the restart of the prometheus daemon